ros/angel_msgs/srv/QueryTaskGraph.srv

```diff
@@ -1,6 +1,12 @@
-# Get the full task graph for the current task.
+# Get all of the task graphs for the current tasks.
+
+# This field is required b/c the ROS TCP bridge cannot handle deserializing an
```

Comment:
I love your addition of descriptions! :100: Chesterton's fence will be harder to move.

Reply:
Haha yeah, I actually tried to remove the `test` field and it broke everything. So I should have left that description the last time we saw that error.